raspbian/applications/rpi-update.md

```diff
@@ -0,0 +1,35 @@
+# rpi-update
+
+`rpi-update` is a command line application that will update your Raspbian kernel and VideoCore firmware to the latest pre-release versions.
+
+**WARNING: Pre-release versions of software are not guaranteed to work. You should not use `rpi-update` on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process.**
+
+The `rpi-update` script is supplied by a third party, "Hexxeh", and also supported by Raspberry Pi engineers. The script source can be found on Github at https://github.com/Hexxeh/rpi-update
+
+## What it does
+
+`rpi-update` will download the latest pre-release version of the linux kernel, its matching modules, device tree files, along with the latest versions of the VideoCore firmware. It will then install these files to the boot partition of the SD card, overwriting any previous versions. 
```

Comment:
As I mentioned in an earlier comment, this also updates the userland libraries and applications, which means that the root partition gets updated too (which is also where the kernel modules live).
Probably not worth mentioning that the userland parts get updated, but I think it might be worth removing the "boot partition" distinction in case a user thinks this means they _only_ need to backup / restore their boot partition in order to "recover" from `rpi-update`?
Perhaps "It will then install these files to the relevant locations on your SD card, overwriting any previous versions." :shrug: